### PR TITLE
refactor(auction-results): removes usage of modal; clean up

### PR DESCRIPTION
**Pull request by dzucconi**
Currently going through great pains to remove our remaining legacy Force modal usage. This appears to be the last usage of the deprecated Palette Modal. After this is merged I'll release a canary with that removed to see.

### CODE REVIEW

File: src/Apps/Artist/Routes/AuctionResults/Components/MarketStats.tsx [hunk 1]
```diff
@@ -70,7 +73,7 @@ export const MarketStats: React.FC<MarketStatsProps> = ({
   )
 
   return (
-    <Box mb={[4, 12]} mt={[0, 6]}>
```
Comment (dzucconi) [inline src/Apps/Artist/Routes/AuctionResults/Components/MarketStats.tsx:73]: External margins.

File: src/Apps/Artist/Routes/AuctionResults/Components/MarketStatsInfoButton.tsx [hunk 2]
```diff
@@ -1,91 +1,106 @@
...
+
 export const MarketStatsInfoButton: React.FC<MarketStatsInfoButtonProps> = ({
   onClick,
 }) => {
   const [showModal, setShowModal] = useState(false)
 
   return (
     <>
-      <InfoCircleIcon
+      <Clickable
```
Comment (dzucconi) [inline src/Apps/Artist/Routes/AuctionResults/Components/MarketStatsInfoButton.tsx:24]: Wasn't wrapped in a `Clickable`

**Comment by dzucconi** [comment]
@olerichter00 I'm going to merge on green to see how this shakes out in Palette but let me know if I'm stepping on anything here as I think there's been some movement on these pages recently.

**Review by olerichter00 — APPROVED** [review]
Looks good 🌟

**Pull request merged**